readthedocs/projects/tasks.py

```diff
@@ -1379,6 +1385,32 @@ def warn(self, msg):
     delete_queryset.delete()
 
 
+def clean_build(version_pk):
+    """Clean the files used in the build of the given version."""
+    version = Version.objects.get_object_or_log(pk=version_pk)
+    if (
+        not version or
+        not version.project.has_feature(Feature.CLEAN_AFTER_BUILD)
+    ):
+        log.info(
+            'Skipping build files deletetion for version: %s',
+            version_pk,
+        )
+        return False
+    del_dirs = [
+        os.path.join(version.project.doc_path, dir_, version.slug)
+        for dir_ in ('checkouts', 'artifacts', 'envs', 'conda')
```

Comment:
Reminder to remove `artifacts` from here for now. We should comment saying that we will add it back once we aren't syncing to the webs.